pegasus/sites.v3/hourofcode.com/public/events/organizations.haml

```diff
@@ -5,17 +5,16 @@ nav: events_nav
 ---
 -SOLR = Solr::Server.new(host:CDO.solr_server)
 
--by_company = {}
+-company_event_counts = Hash.new {|h,k| h[k] = 0}
```

Comment:
Yep. It started that way when I made the default an array (where that funniness) is necessary. Nice catch. Done.